server/models/user-page.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+// we need a user-pages abstraction so that async scraper jobs
+// can separately handle creating the visit and creating the
+// user page. someday this might also be used by the client-facing API layer.
+
+var underscored = require('underscore.string').underscored;
+var uuid = require('uuid');
+
+var postgres = require('../db/postgres');
+var elasticsearch = require('../db/elasticsearch');
+var log = require('../logger')('server.models.user-page');
+
+var _verbose = function() {
+  var logline = [].join.call(arguments, ', ');
+  log.verbose(logline);
+};
+
+var userPage = {
+  _onFulfilled: function _onFulfilled(msg, callback, results) {
+    _verbose(msg);
+    callback(null, results);
+  },
+  _onRejected: function _onRejected(msg, callback, err) {
```

Comment:
totally agree, but this whole `.done(foomodel._onFulfilled.bind(null ... ), foomodel._onRejected.bind(null...)` thing needs work anyway; I predict it vanishes sooner than later